doc/contribute.rst

```diff
@@ -0,0 +1 @@
+../CONTRIBUTE.rst
```

Comment:
Note: This is a softlink. I could not link the CONTRIBUTE.rst file into the docs unless there was a link from inside the doc folder.